index.md

```diff
@@ -4,6 +4,6 @@ layout: default
 # To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
 
 ---
-Welcome to our Documentation site. How about a visit to [GitHub Docs](https://help.github.com/)
+Welcome to our Documentation site. How about a visit to [GitHub Docs](https://help.github.com/setupgit)
```

Comment:
Welcome to our Documentation site. How about a visit to [GitHub Docs](https://help.github.com/articles/set-up-git/)